tests/integration/ledger/mocks/AccountRetriever.go

```diff
@@ -0,0 +1,125 @@
+// Code generated by mockery v2.15.0. DO NOT EDIT.
+
+package mocks
+
+import (
+	client "github.com/cosmos/cosmos-sdk/client"
+	mock "github.com/stretchr/testify/mock"
+
+	types "github.com/cosmos/cosmos-sdk/types"
+)
+
+// AccountRetriever is an autogenerated mock type for the AccountRetriever type
+type AccountRetriever struct {
+	mock.Mock
+}
+
+// EnsureExists provides a mock function with given fields: clientCtx, addr
+func (_m *AccountRetriever) EnsureExists(clientCtx client.Context, addr types.AccAddress) error {
+	ret := _m.Called(clientCtx, addr)
+
+	var r0 error
+	if rf, ok := ret.Get(0).(func(client.Context, types.AccAddress) error); ok {
```

Comment:
rename r0 to `err`. What is `rf`?

Reply:
@fedekunze this file us autogenerated trough mockery. I don't see a need to change this. We follow the same pattern on ethermint -> https://github.com/evmos/ethermint/blob/main/rpc/backend/mocks/client.go